sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseDataDto.java

```diff
@@ -390,6 +403,28 @@
 	private Date followUpUntil;
 	private boolean overwriteFollowUpUntil;
 
+	@HideForCountriesExcept(countries = "ch")
```

Comment:
Just a suggestion, but you could use `CountryHelper.COUNTRY_CODE_SWITZERLAND` for easier tracing later if needed instead of the "hardcoded" country code.